From 81bb97672d4aee8405e36162602de79345e848b7 Mon Sep 17 00:00:00 2001
From: Bin Han <bin.b.han@intel.com>
Date: Wed, 21 Oct 2015 17:55:19 +0800
Subject: [PATCH] camera: add BTNS camera driver path

On BTNS r6_legacy branch, the external camera driver folder
is camera_icgotm instead of camera and CONFIG_ENTERNAL_CAMERA_ICGOTM
will be defined instead of CONFIG_CONFIG_EXTERNAL CAMERA, thus we
also need to check CONFIG_ENTERNAL_CAMERA_ICGOTM in Makefile and
Kconfig.

Change-Id: I38f8cc2c0103bfa6788537e32583a133ded6dcae
Original-Tracked-On: #H1504070375
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-7449
Signed-off-by: Bin Han <bin.b.han@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/80642
Build: Dessert M BXT RVP (SC)
Approver: Tian Shu Qiu <tian.shu.qiu@intel.com>
Validation-Android: Wei Tao <wei.tao@intel.com>
Build: Dessert M COHO (SC)
Integrator: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
---
 drivers/external_drivers/camera/Kconfig  | 5 +++++
 drivers/external_drivers/camera/Makefile | 1 +
 2 files changed, 6 insertions(+)

diff --git a/drivers/external_drivers/camera/Kconfig b/drivers/external_drivers/camera/Kconfig
index e51f2f1..c2e8418 100644
--- a/drivers/external_drivers/camera/Kconfig
+++ b/drivers/external_drivers/camera/Kconfig
@@ -1,3 +1,8 @@
 if EXTERNAL_CAMERA
 source drivers/external_drivers/camera/drivers/Kconfig
 endif
+
+if EXTERNAL_CAMERA_ICGOTM
+# for BTNS camera drivers
+source drivers/external_drivers/camera/drivers/Kconfig
+endif
diff --git a/drivers/external_drivers/camera/Makefile b/drivers/external_drivers/camera/Makefile
index a175427..cbb260e 100644
--- a/drivers/external_drivers/camera/Makefile
+++ b/drivers/external_drivers/camera/Makefile
@@ -1 +1,2 @@
 obj-$(CONFIG_EXTERNAL_CAMERA) += drivers/
+obj-$(CONFIG_EXTERNAL_CAMERA_ICGOTM) += drivers/
-- 
1.9.1

